core/web_admin.py

```
@@ -0,0 +1,1247 @@
+        self._login_data = {
+            "replace_existing": True,
+            "need_password": False,
+            "created_at": datetime.now().isoformat(timespec="seconds"),
+        }
+        return {"message": "已进入重新登录流程，当前已登录账号会保留到新账号登录成功。"}
+
+    async def runtime_check(self) -> dict[str, Any]:
+        self.plugin.forwarder._stopping = False
+        operation = self._new_runtime_operation(
+            "立即抓取发送",
+            "已排队，准备强制抓取频道更新。",
```

Ruff (0.15.20)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)